app/controllers/sdg_management/local_targets_controller.rb

```diff
@@ -1,19 +1,16 @@
 class SDGManagement::LocalTargetsController < SDGManagement::BaseController
   include Translatable
 
-  LocalTarget = ::SDG::LocalTarget
+  load_and_authorize_resource class: "SDG::LocalTarget"
 
   def index
-    @local_targets = LocalTarget.all.sort
+    @local_targets = @local_targets.sort
   end
 
   def new
-    @local_target = LocalTarget.new
   end
 
   def create
-    @local_target = LocalTarget.new(local_target_params)
```

Comment:
I think we need to keep this line :thinking:; otherwise we are not filtering `params` and so are vulnerable to mass assignment.

Correct me if I'm wrong!

Reply:
I was wrong :smile:.

https://github.com/CanCanCommunity/cancancan/wiki/Strong-Parameters